src/Components/Signup.jsx

```diff
@@ -73,7 +73,7 @@ class Signup extends Component {
       }
     }
     if (this.state.errorMessage !== '') {
```

Comment:
Javascript returns false if you have an empty string.
You can shorten this by saying `if(this.state.errorMessage)`